### ISSUE TITLE
Any plans for receiving SMS and MMS?

### ISSUE THREAD

**Issue by BedHead911** [kontalk/androidclient#726]
Kontalk seems like a good alternative to Signal. I was wondering if there are plans to make this app function as a default messaging app. It would be a good reason for me to keep using this app while I convince my friends to switch over.

**Comment by daniele-athome** [comment]
Hi! Not at the moment sorry, maybe in the future but currently it's not on the project roadmap. Of course any help is welcomed!

**Comment by legovini** [comment]
Just to add another opinion, I personally dislike the feature, as it breaks the "everything in Kontalk is encrypted" paradigm. I find it just out of its scope.

For SMS messaging I use SMSsecure (now Silence) that also allows for encrypted messages.

**Comment by daniele-athome** [comment]
I agree with that philosophy too, but I don't want to exclude it totally either. I think the optimal approach in this case would be something like an external module (it would require some sort of plugin API though).

**Comment by legovini** [comment]
From one side I'd like it better, from the other side a plugin API could be a very weak point of the app (malicious plugins, bugs...). So I'm still not sure about the idea, but leaving the possibility open is not bad either.

**Comment by BedHead911** [comment]
I'm just a regular user who supports privacy. From my point of view, I like user friendliness and UI of Kontalk but my contact list has nobody in it. The fact that Kontalk is open source and doesn't depend on google makes it superior to Hangout or Signal in my opinion. But it difficult to convince others to switch and more difficult for me to keep this app installed when nobody I know uses it.

I just wanted to share my point of view amd wanted to say that I'm glad you guys are considering this.

**Comment by keverets** [comment]
@mjg59 mentioned in the comments of his blog post that this was a blocker to adoption (at http://mjg59.dreamwidth.org/42728.html#comments ).

It would improve the user experience to be able to use Kontalk as the default messaging application with graceful fall-back for users that have not yet converted.
